25-minimize.ks

```diff
@@ -18,8 +23,10 @@ systemctl disable sshd.service
 
 # This seems to cause 'reboot' resulting in a shutdown on certain platforms
 # See https://tickets.puppetlabs.com/browse/RAZOR-100
-echo " * remove intel mei modules"
-rm -rf /lib/modules/*/kernel/drivers/misc/mei
+# ^^ URL above doesn't exists anymore.
+# Keeping here for the records.
+#echo " * remove intel mei modules"
+#rm -rf /lib/modules/*/kernel/drivers/misc/mei
 
 # See https://bugzilla.redhat.com/show_bug.cgi?id=1335830
```

Comment:
This was also for RHEL 7 and I question I recall we no longer support kexec, but on the other hand I know the amdgpu firmware is big. It may still help to reduce the size of the image to remove these.